lib/nodeMapper.ts

```diff
@@ -192,7 +192,7 @@ export const mapNodes = (
         dragHandle: `#${cKey}-dragHandle`,
         position: {
           x: offset(cKey, nodesMap.components),
-          y: 650 + (max - 1) * 76,
```

Comment:
Remove the lower throughput edge and pull the data source widgets back up a bit.